src/kibana/components/doc_table/doc_table.less

```diff
@@ -7,6 +7,12 @@ doc-table {
   margin: 5px;
   .flex(1, 1, 100%);
 
+  tr.discover-table-row  {
+    td:not(.discover-table-timefield):not(.ng-scope) {
```

Comment:
I'm just generally cautious about overly specific selectors. I'd rather tag the cells we want to apply the class to than use multiple `:not()` selectors.